src/client/Microsoft.Identity.Client/Platforms/Android/Broker/BrokerRequest.cs

```diff
@@ -65,7 +65,17 @@ public Uri RedirectUri
             AcquireTokenInteractiveParameters acquireTokenInteractiveParameters)
         {
             BrokerRequest br = FromAuthenticationParameters(authenticationRequestParameters);
-            br.Prompt = acquireTokenInteractiveParameters.Prompt.PromptValue.ToUpperInvariant();
+            var prompt = acquireTokenInteractiveParameters.Prompt.PromptValue;
```

Comment:
nit: you can compare Prompt objects directly as they have `==` overridden. Pretty obscure, not sure who decided that :(